mp/src/game/shared/momentum/fx_mom_shared.cpp

```diff
@@ -181,16 +183,37 @@ void FX_FireBullets(int iEntIndex, const Vector &vOrigin, const QAngle &vAngles,
         lagcompensation->StartLagCompensation(pPlayer, pPlayer->GetCurrentCommand());
 #endif
 
-    for (int iBullet = 0; iBullet < pWeaponInfo->m_iBullets; iBullet++)
-    {
-        RandomSeed(iSeed); // init random system with this seed
+    int iTotalBullets = pWeaponInfo->m_iBullets;
+    bool bNoSpread = false;
 
-        // Get circular gaussian spread.
+    if (iTotalBullets > 1)
+        bNoSpread = mom_use_fixed_spread.GetBool();
```

Comment:
You can combine this into the assignment:

`bool bNoSpread = iTotalBullets > 1 && mom_fixed_spread.GetBool();`